### PR TITLE
Development [HERA-202] [End Survey] Base functionality

### PR DESCRIPTION
**Pull request by RuslanSkiraRaccoongang**
**Description:** Development [HERA-202] [End Survey] Base functionality
	- user see the questions with options.
	- user has 2 slide windows with questions.
	- staff static functionality does not implement in this commit.

**Youtrack:HERA-203** https://youtrack.raccoongang.com/issue/HERA-203

**Reviewers:**
- [ ] @sendr 
- [ ] @UvgenGen 
- [ ] @dyudyunov 

**Merge checklist:**
- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Documentation in source code updated
- [ ] All related Confluence documentation is updated (including deployment documentation)
- [ ] Commits are squashed

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

### CODE REVIEW

File: hera_xblocks/end_survey/end_survey/end_survey.py [hunk 1]
```diff
@@ -100,7 +105,9 @@ def vote(self, data, sufix=''):
 
         self.user_result = data
         self.result_summary.append(data)
-        self.user_count += 1
+        # TODO: need do not add to user count if the same user and disable button. 
+        if self.user_result:
```
Comment (sendr) [inline hera_xblocks/end_survey/end_survey/end_survey.py]: `if not self.user_result`

File: hera_xblocks/end_survey/end_survey/static/html/end_survey_user.html [hunk 2]
```diff
@@ -1,25 +1,61 @@
...
+                        <div class="button-wrapper">
+                            <button class="button-next" type="button">Submit Survey</button>
+                        </div>
+                    </div>
+                </div>
+            </form>
+        </div>
+        <div class="single-item">
+                    <form class="survey-form">
+                <div class="end_survey_block">
```
Comment (sendr) [inline hera_xblocks/end_survey/end_survey/static/html/end_survey_user.html]: something wrong with indentations

File: hera_xblocks/end_survey/end_survey/static/html/end_survey_user.html [hunk 3]
```diff
@@ -1,25 +1,61 @@
...
+                    <div class="servey-questions">
+                            {% dictvalue user_result confidence.questionText as result %}
+                            <div class=row>
```
Comment (sendr) [inline hera_xblocks/end_survey/end_survey/static/html/end_survey_user.html]: indentations

File: hera_xblocks/end_survey/end_survey/static/js/src/end_survey.js [hunk 4]
```diff
@@ -1,15 +1,45 @@
...
+    
+    $(".button-next").click((e) => {
+        $('.slidebar-wrapper').slick('slickNext')
+    });
     
     $(".button-submit").click((e)=> {
         $('.button-submit').prop('disabled', true);
-        const userForm = $('#end_survey_form').serializeArray();
+        const answer = $questionForm.serializeArray();
+        var t = {};
```
Comment (sendr) [inline hera_xblocks/end_survey/end_survey/static/js/src/end_survey.js]: rename it please

File: lms/djangoapps/hera/templatetags/dictvalue.py [hunk 5]
```diff
@@ -0,0 +1,13 @@
+from django import template
+from urllib import quote_plus
+
+register = template.Library()
+
+@register.assignment_tag
+def dictvalue(answers, key):
+    """Function accepts dictionay and key return value."""
```
Comment (sendr) [inline lms/djangoapps/hera/templatetags/dictvalue.py]: typo

File: lms/djangoapps/hera/templatetags/dictvalue.py [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+    value = ''
```
Comment (sendr) [inline lms/djangoapps/hera/templatetags/dictvalue.py]:
```suggestion
    for answer in answers:
        if answer['name'] == key:
            return answer['value']
    return ''
```

File: hera_xblocks/end_survey/end_survey/end_survey.py (hunk 1 above)
Comment (RuslanSkiraRaccoongang) [reply hera_xblocks/end_survey/end_survey/end_survey.py]: Change to if not self.user_result

File: hera_xblocks/end_survey/end_survey/static/html/end_survey_user.html (hunk 2 above)
Comment (RuslanSkiraRaccoongang) [reply hera_xblocks/end_survey/end_survey/static/html/end_survey_user.html]: Moved to the right position.

File: hera_xblocks/end_survey/end_survey/static/html/end_survey_user.html (hunk 3 above)
Comment (RuslanSkiraRaccoongang) [reply hera_xblocks/end_survey/end_survey/static/html/end_survey_user.html]: Fixed. With class=row -> class="row"

File: hera_xblocks/end_survey/end_survey/static/js/src/end_survey.js (hunk 4 above)
Comment (RuslanSkiraRaccoongang) [reply hera_xblocks/end_survey/end_survey/static/js/src/end_survey.js]: Done

File: lms/djangoapps/hera/templatetags/dictvalue.py (hunk 5 above)
Comment (RuslanSkiraRaccoongang) [reply lms/djangoapps/hera/templatetags/dictvalue.py]: dictioany - >dictionary

File: lms/djangoapps/hera/templatetags/dictvalue.py (hunk 6 above)
Comment (RuslanSkiraRaccoongang) [reply lms/djangoapps/hera/templatetags/dictvalue.py]: done

**Pull request merged**